apps/data-pipeline/ap-exams-importer/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { exit } from "node:process";
+
+import { database } from "@packages/db";
+import { apExams } from "@packages/db/schema";
+
+import { conflictUpdateSetAllCols } from "@packages/db/utils";
+
+import apExamNames from "./names";
+
+async function main() {
+  const url = process.env.DB_URL;
+  if (!url) throw new Error("DB_URL not found");
+  const db = database(url);
+
+  await db
+    .insert(apExams)
+    .values(
+      Object.entries(apExamNames).map(([k, v]) => ({
```

Comment:
When destructuring key-value tuples from `Object.entries`, the key and value can be named whatever you want. So you could theoretically call them `id` and `officialName`, then use the object initializer shorthand in the object returned by `Array.map`.

I don't know if this is actually any more concise though, I just like using the objinit shorthand.

Reply:
Don't really have an opinion; I'll do that.